### ISSUE TITLE
Visual Editor

### ISSUE THREAD

**Issue by assadnazar** [OrchardCMS/OrchardCore#11051]
Are you guys planning for a visual editor like Elementor for Wordpress?

**Comment by ns8482e** [comment]
@assadnazar Orchard has live preview

**Comment by assadnazar** [comment]
> 

Yeah I can see that. Only query was if Orchard CMS is planning for a visual editor module?

**Comment by ns8482e** [comment]
@assadnazar interesting.

Visual Editors are normally Theme first approach where you select the theme first and then you build content for that theme.

Orchard Core is using different approach- it's content first - meaning you can run it as headless or decoupled scenarios where theme is not required.

However if you'd like have side by side live preview feedback on different devices, you can use `DevicePreview` module https://github.com/surevelox/OrchardCore.Modules/blob/master/DevicePreview/README.md

**Comment by hishamco** [comment]
Thanks for sharing Niraj, I was planning to do something quite similar in OCC https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/issues/18

**Comment by sebastienros** [comment]
No plan for that. More like a plan to not do that actually.
